docs/credits.md

```diff
@@ -1518,6 +1541,9 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | process                                           | Roman Shtylman                                                            | [https://github.com/shtylman/node-process](https://github.com/shtylman/node-process)                                                                                                                                        | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | process-nextick-args                              |                                                                           | [https://github.com/calvinmetcalf/process-nextick-args](https://github.com/calvinmetcalf/process-nextick-args)                                                                                                              | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | progress                                          | TJ Holowaychuk                                                            | [https://github.com/visionmedia/node-progress](https://github.com/visionmedia/node-progress)                                                                                                                                | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
+| prometheus-net.AspNetCore                         | andrasm,qed-,lakario,sandersaares                                         | [https://github.com/prometheus-net/prometheus-net](https://github.com/prometheus-net/prometheus-net)                                                                                                                        | [MIT](https://www.nuget.org/packages/prometheus-net.AspNetCore/4.2.0/license)                                       |
+| prometheus-net.DotNetRuntime                      | James Luck                                                                | [https://github.com/djluck/prometheus-net.DotNetRuntime](https://github.com/djluck/prometheus-net.DotNetRuntime)                                                                                                            | [Undefined]()                                                                                                       |
```

Comment:
This one is MIT, again, curious why it didn't work automatically. :) https://github.com/djluck/prometheus-net.DotNetRuntime/blob/master/LICENSE.txt

Reply:
The package is missing license info, added a special case https://github.com/OctopusDeploy/DependencyScanner/pull/54